packages/expo-web-browser/src/WebBrowser.types.ts

```diff
@@ -7,6 +7,7 @@ export type OpenBrowserOptions = {
   browserPackage?: string;
   enableBarCollapsing?: boolean;
   showTitle?: boolean;
+  controlsColor?: string;
```

Comment:
```suggestion
  controlsColor?: string; // iOS only
```
We usually add a comment if the property is available only on specific platforms 😉 Don't forget to rebuild JS files after this change.